packages/frontend/src/components/common/layout/Footer.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+interface FooterProps {}
+
+const Footer: React.FC<FooterProps> = () => {
+  return (
+    <div className="opacity-50 w-full mt-8">
+      <div className="w-full text-center p-4">Copyright Team 4</div>
```

Comment:
Should it be copyright when the project is licensed under MIT?

Reply:
It was just random text i chucked in 😅 but it led me on an interesting search.

The MIT license is about granting the right to use our copyrighted materials. So we still hold the copyright (which we do implicitly as we created the work)

https://softwareengineering.stackexchange.com/questions/117572/mit-and-copyright

Reply:
ah very interesting 😄